Common/FeatureFlags.swift

```diff
@@ -81,6 +82,13 @@ struct FeatureFlagConfiguration: Decodable {
         #else
         self.lyumjevInsulinModelEnabled = true
         #endif
+
+        // Swift compiler config is inverse, since the default state is enabled.
+        #if LYUMJEV_INSULIN_MODEL_DISABLED
```

Comment:
Shouldn’t this line reference Afrezza rather than Lyumjev?

Reply:
Fixed, thanks!